pylint/checkers/typecheck.py

```diff
@@ -1780,7 +1780,7 @@ def visit_compare(self, node):
             return
 
         op, right = node.ops[0]
-        if op in ["in", "not in"]:
```

Comment:
I don't know why it was not caught in the CI before, but I think this warning is too opinionated, we can disable it in the pylintrc instead. There's a discussion to replace it by ``use-set-for-membership`` as sets are faster for membership, which is what we should use here but clearly not related to this MR. See #4776 / https://github.com/PyCQA/pylint/pull/4841, if you're interested about this issue :)

Reply:
What should I do for now? I can't push any commits without changing these..

Reply:
Please disable the check by adding ``consider-using-tuple``after ``disable=`` inside the ``pylintrc`` file (at the root of the repository).

Reply:
Wouldn't that disable the check even after the updates of #4841 have been added?

Reply:
I would not mind if it's disabled it's very opinionated. It will permit to keep working on this and I'll check the root cause meanwhile :)